.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: main
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  # skip ci if the last commit contains the appropriate tag
+  skip-commit:
+    name: Conditionally skip ci
+    runs-on: ubuntu-latest
+    steps:
+      - if: "contains(github.event.head_commit.message, '[skip-ci]')
+          || contains(github.event.head_commit.message, '[skip ci]')
+          || contains(github.event.head_commit.message, '[ci-skip]')
+          || contains(github.event.head_commit.message, '[ci skip]')"
```

Comment:
Oh, this is because it allows for either a space or a dash for each variant.

To be honest though, I don't think I've ever actually even used this job. I just keep it around in most of my workflows out of habit. 

I think I'll just remove it unless there's a need for it at some point.

Reply:
Oh wow, I feel dumb! I don't think I noticed the space vs. hyphenated distinction amidst my review! Thanks for pointing it out. :joy:

Reply:
Oh wow, I feel dumb! I don't think I noticed the space vs. hyphenated distinction during my review! Thanks for pointing out the distinction. I don't mind its presence either way, but it's nice to know this exists. :joy: